emukit/quadrature/kernels/integration_measures.py

```diff
@@ -42,8 +44,28 @@ def get_box(self) -> List[Tuple[float, float]]:
         """
         raise NotImplementedError
 
+    @property
+    def can_sample(self):
+        """True if combined with IntegrationMeasureSampler"""
+        raise NotImplementedError
+
 
-class UniformMeasure(IntegrationMeasure):
+class IntegrationMeasureSampler:
+    """Augments the integration measure with sampling capabilities. This might be needed for some candidate point
+    collection schemes."""
+
+    def get_samples(self, num_samples: int, context_manager: ContextManager=None) -> np.ndarray:
+        """
+        samples from the probability distribution defined by the integration measure.
+        :param num_samples: number of samples
+        :param context_manager: The context manager Contains variables to fix. If a context is given, this method
```

Comment:
I feel like "The context manager Contains variables to fix." needs rephrasing. Something like "The context manager that contains variables to fix and the values to fix them to."

Reply:
done